sdk/storage/azure-storage-blob/tests/perfstress_tests/download.py

```diff
@@ -0,0 +1,35 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure_devtools.perfstress_tests import get_random_bytes, WriteStream
+
+from ._test_base import _ContainerTest
+
+
+class DownloadTest(_ContainerTest):
+    def __init__(self, arguments):
+        super().__init__(arguments)
+        blob_name = "downloadtest"
+        self.blob_client = self.container_client.get_blob_client(blob_name)
+        self.async_blob_client = self.async_container_client.get_blob_client(blob_name)
+
+    async def global_setup(self):
+        await super().global_setup()
+        data = get_random_bytes(self.args.size)
+        await self.async_blob_client.upload_blob(data)
+
+    def run_sync(self):
+        download = WriteStream()
```

Comment:
Can `WriteStream` be reused?

Reply:
I've added a reset to do this, so we can exclude this from the timed function